src/Api/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -38,7 +39,7 @@ public static void AddSwagger(this IServiceCollection services, GlobalSettings g
                         TokenUrl = new Uri($"{globalSettings.BaseServiceUri.Identity}/connect/token"),
                         Scopes = new Dictionary<string, string>
                         {
-                            { "api.organization", "Organization APIs" },
+                            { Scopes.ApiOrganization, "Organization APIs" },
```

Comment:
I don't think this string is used anywhere else than here and since it's just a description I don't think it would be to bad for it to change in the future. I'd rather not clutter constants to much until it's used in more than one location.